lightspeed/modules/proc_view-logs.adoc

```diff
@@ -0,0 +1,26 @@
+:_content-type: PROCEDURE
+
+[id="view-logs_{context}"]
+
+= Viewing the audit logs
+
+The Ansible VS Code extension records all Ansible Lightspeed operations in an audit log for future use. Each interaction is recorded with a time stamp, the type of action performed, details of the requested task, and other relevant information.
+
+[NOTE]
+====
+If you close VS Code, the log history is deleted. 
+====
+
+.Procedure
+
+. Open Visual Studio Code.
```

Comment:
This is a nit, but the full spelling "Visual Studio Code (VS Code)" should come at the first mention, which occurs earlier.